src/tike/ptycho/solvers/__init__.py

```diff
@@ -1,10 +1,12 @@
 """Contains different solver implementations."""
 
-# from .admm import admm
+from .admm import admm, admm1
 from .combined import combined
 from .divided import divided
 
 __all__ = [
+    "admm",
+    "admm1",
```

Comment:
what is admm1?

Reply:
The one where `recover_nearplane` is set to False. Ignore it, it's for development purposes.